mealpy/mealpy.py

```diff
@@ -68,6 +68,9 @@ def login(self, user, password):
             'password': password,
         }
         request = self.session.post(LOGIN_URL, data=json.dumps(data))
```

Comment:
This change breaks the program. Not having the json.dumps causes Mealpal to return a bad request message for all reservations.